include/openssl/evp.h

```diff
@@ -511,6 +511,8 @@ OPENSSL_EXPORT int EVP_PKEY_print_params(BIO *out, const EVP_PKEY *pkey,
 // function that results in a key suitable for use in symmetric
 // cryptography.
 
+// Deprecated constant as used by deprecated EVP_BytesToKey() which
+// cannot change
 #define PKCS5_SALT_LEN 8
```

Comment:
I believe CI job requires this comment to begin with the word "PKCS5_SALT_LEN". (I did verify that the "diff" I provided succeed the check)